EpiAware/src/latentmodels/autoregressive.jl

```diff
@@ -0,0 +1,46 @@
+@doc raw"
+The autoregressive (AR) model struct.
+"
+@with_kw struct AR{D <: Priors, S <: Distribution, I <: Priors} <: AbstractLatentModel
```

Comment:
I like this use of `Parameters.@with_kw`. Happy to have `Parameters.jl` as a dep.